src/features/home/Moonpots/Moonpots.js

```diff
@@ -55,16 +49,21 @@ const Moonpots = ({ selected }) => {
     }
   }, [dispatch, address]);
 
+  useEffect(() => {
+    if (sortKey !== filterConfig.sortKey || sortDir !== filterConfig.sortDir) {
+      setFilterConfig({ ...filterConfig, sortKey, sortDir, version: filterConfig.version + 1 });
```

Comment:
The version should only be updated when we change the structure of the data stored in localstorage
e.g. if we add a field, then the users saved copy wouldn't have this field, so if we tried to access it in the app, the app would crash

When the app loads the config from localstorage, it checks if the apps version is higher than the user's stored version, and reset the config to default if so.

Reply:
Removed this part and I am only changing the key and direction now.